pkg/api/types.go

```diff
@@ -673,6 +673,9 @@ type GlusterfsVolumeSource struct {
 	// Required: EndpointsName is the endpoint name that details Glusterfs topology
 	EndpointsName string `json:"endpoints"`
 
+	// Optional: Servers is the gluster server list
+	EndpointsNameSpace *string `json:"endpointsns,omitempty"`
```

Comment:
That's a good question. Given one can just look up the endpoints in DNS, why does it need to be?